src/brevitas/graph/equalize.py

```diff
@@ -1764,6 +1763,9 @@ def __init__(
         self.return_rewriters = return_rewriters
         self.sdpa_regions = sdpa_regions
         self.expansion_step = expansion_step
+        self.delay_rewriters = delay_rewriters
+        if not self.delay_rewriters:
+            assert return_rewriters, "If rewriters are not applied immediately, they must be returned and applied at a second moment"
```

Comment:
I would make it clear for the user which flags are involved:
```suggestion
            assert return_rewriters, "If  `delay_rewriters=True`, rewriters are not applied immediately, so they must be returned, by setting `return_rewriters=True`, to be applied at a later stage."
```